src/turndown.js

```diff
@@ -488,7 +488,7 @@ turndownService.addRule("multiple_choice_question", {
     if (node.nodeName === "DIV" && node.getAttribute("class")) {
       if (
         node.getAttribute("class") === "problem_question" &&
-        node.getElementsByClassName("problem_radio_input")
+        node.getElementsByClassName("problem_radio_input").length > 0
```

Comment:
There's a `child.getElementsByClassName("problem_radio_input")` in the `replacement:` section below, maybe add `.length > 0` there too just in case?

Reply:
Nothing with no results on that class search should get through the filter, but I added this anyway because it's a sensible precaution.